lib/server.js

```diff
@@ -1052,6 +1055,10 @@ WalletService.prototype.createTx = function(opts, cb) {
           excludeUnconfirmedUtxos: !!opts.excludeUnconfirmedUtxos,
         });
 
+        if (!self.clientVersion || _.startsWith(self.clientVersion, 'bwc-0.0.')) {
+          txp.version = '1.0.1';
```

Comment:
I agree in general terms but in this particular case we will end up with hybrid schema as client code makes use of the version field.